arch/configure.defaults

```diff
@@ -1975,6 +1975,47 @@ LIB_BUNDLED     = \
                      $(WRF_SRC_ROOT_DIR)/frame/pack_utils.o
 
 #insert new stanza here
+################################################## #########
+    #ARCH   Linux   aarch64,gnu OpenMPI #serial smpar dmpar dm+sm
```

Comment:
I think the next line should be just `#`.
Also all lines below the stanza should not have any indentation.
Without these corrections I was not able to build it.